### DISCUSSION TITLE
For Spanish dictionary. `tengo` is a valid spanish word

### DISCUSSION THREAD

**Discussion by dhdez-blife** [barrust/pyspellchecker#172] [Q&A]
I'm working in build a chatbot in spanish. I looked your package to apply corrections in the phrases. When I tested with my phrases it says the word `tengo` is a mistake. In our vocabulary this is a valid word. Would you like to apply the respective that the word is valid to avoid to call corrections? please.
UPDATED: March 12th, 2024.
New words to be added:
- estoy
- buscando

**Comment by barrust** [comment]
This should be fixed in the next version.
